action.yml

```diff
@@ -8,5 +8,5 @@ inputs:
     description: 'domains exempt from scanning'
     required: false
 runs:
-  using: 'node14'
+  using: 'node12'
```

Comment:
Could you clarify why you use a lower version?

Reply:
This is what actually Github supports. I was thinking it supports node14 and tried to update from 12 but then got an error, so just reverting this back.

Reply:
Got it. Thank you!